### ISSUE TITLE
Autonomous Modes

### ISSUE THREAD

**Issue by Muirrum** [AndromedaOne/2020Code#8]
Should probably be split into different issues, but this is here for documentation from my meeting w/ R&S

**Reference ASD Slide 10**

## Five Point Drive
- Start robot with bumper 1" over initiation line
- move backwards 1'

## Three balls - known starting position - centered in front of port
- aimed at the inner port
- either behind or in front of initiation line
- no vision, shoot very fast
- move back 1'
- if back bumper is on line, make uturn and move forward (avoid robots directly behind us)

## Three balls - known starting position - centered on the trench
- shoot for hexagon goal (outer not lower)
- facing the port
- move 1'

## Pick up power cells from trench
**Dependent on intake design**
- possibly after previous auto, drive down trench with intake out and refill balls
- go to known trench shooting position and shoot 5 balls into outer port
- also should be able to work on opposing team's trench (low priority)

## Shield Generator
- at some point R&S wants us to get at least some balls from the Shield Generator after shooting
- only after the previous ones get reliable
